### PR TITLE
workflow 7.x | follow-up based on SME comments for LRDOCS-7725

### PR DESCRIPTION
**Pull request by rbohl**
As per comments on https://github.com/jrhoun/liferay-learn/pull/1113
https://issues.liferay.com/browse/LRDOCS-7725
cc @lipusz

### CODE REVIEW

File: docs/dxp/latest/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/connecting-to-elasticsearch.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ If Elasticsearch is [installed and running](./installing-elasticsearch.md), star
 
 ![An active connection is displayed in the Search administrative panel.](./getting-started-with-elasticsearch/images/01.png)
 
-Re-index your search indexes and spell check indexes. Invoke both of these actions in the Index Actions tab of Control Panel &rarr; Configuration &rarr; Search.
+Re-index the search indexes and spell check indexes for each Virtual Instance. Invoke both of these actions in the Virtual Instance's Index Actions tab of Control Panel &rarr; Configuration &rarr; Search.
```
Comment (lipusz) [inline docs/dxp/latest/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/connecting-to-elasticsearch.md:163]: This is not needed. What I wrote applies to the Workflow Metrics indexes only.

File: docs/dxp/latest/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-to-elasticsearch-7.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ Once Liferay is connected with the Elasticsearch cluster, restart Liferay and re
 
 1. Restart Liferay.
 
-1. Re-index the company, system, and spell check indexes. From the Global Menu (![Global Menu](../../../../images/icon-applications-menu.png)), navigate to *Control Panel* &rarr; *Configuration* &rarr; *Search*. Click *Execute* for the *Reindex all search indexes* entry.
+1. Re-index the company, system, and spell check indexes. In each Virtual Instance, from the Global Menu (![Global Menu](../../../../images/icon-applications-menu.png)), navigate to *Control Panel* &rarr; *Configuration* &rarr; *Search*. Click *Execute* for the *Reindex all search indexes* entry.
```
Comment (lipusz) [inline docs/dxp/latest/en/using-search/installing-and-upgrading-a-search-engine/elasticsearch/upgrading-elasticsearch/upgrading-to-elasticsearch-7.md:76]: Not needed.

**Comment by rbohl** [comment]
Ahh right, of course--sorry, context switching made my bran work improperly. Will fix.

**Comment by jrhoun** [comment]
Merged. Thanks.

**Pull request closed without merging**